valgrind/fjalar/dwarf.c

```diff
@@ -58,7 +58,12 @@
 # define ngettext(Msgid1, Msgid2, n) \
   (n == 1 ? Msgid1 : Msgid2)
 //#include "libiberty.h"
+// 2022.07.06 uWX1176834 added startswith macro
```

Comment:
What is "uWX1176834"?  Is that you, or a marker for some commit, or ...?

Reply:
That is me, it is my internal nickname in a company

Reply:
Thanks for the explanation.  Could you please either replace it, or augment it, by your GitHub ID and/or your name?  That will reduce confusion in the future.

Reply:
Done